packages/interactivity/src/init.js

```diff
@@ -21,15 +21,25 @@ export const getRegionRootFragment = ( region ) => {
 	return regionRootFragments.get( region );
 };
 
+function yieldToMain() {
+	return new Promise( ( resolve ) => {
+		// TODO: Use scheduler.yield() when available.
```

Comment:
What about `isInputPending`?

Reply:
Humm. Good question. Since `init` runs at `DOMContentLoaded` it seems like it would be unlikely that there would be input pending at this early point, but it could happen. But if we're already yielding with each `toVdom()` and `hydrate()`, then it seems `schedule.isInputPending()` would only be used to decide whether or not to skip yielding, right?

Reply:
Also, it would seem like `schedule.isInputPending()` would be much more important in the context of actions, since they are more inherently running at the time of user input.

Reply:
I've incorporated `isInputPending()` into #58284 for lazy-hydration. There, since hydration could happen at any point of the lifecycle of the page when a user is potentially doing an input, I think it makes more sense to check for input there.

Reply:
Ok, thanks Weston.